### PR TITLE
fix: CozyDialog Title should have ellipsis with long text

### PR DESCRIPTION
**Pull request by JF-Cozy**
fix bug introduced with https://github.com/cozy/cozy-ui/commit/4bc03f4106ded21b425b92da09e1c30ab23f960d because of display flex

### CODE REVIEW

File: react/CozyDialogs/Readme.md [hunk 1]
```diff
@@ -132,29 +132,41 @@ initialState = {
...
     <DialogComponent
       size={DialogComponent !== ConfirmDialog ? state.size : undefined}
       open={state.modalOpened}
       onClose={handleClose}
-      title={dialogTitles[DialogComponent.name]}
+      title={DialogComponent !== IllustrationDialog
+        ? state.title === "long"
+          ? `${dialogTitles[DialogComponent.name]} - ${content.ada.short}`
+          : dialogTitles[DialogComponent.name]
+        : dialogTitles[DialogComponent.name]
```
Comment (Crash--) [inline react/CozyDialogs/Readme.md:168]: hard to read 
```
DialogComponent !== IllustrationDialog &&  state.title === "long" ? `${dialogTitles[DialogComponent.name]} - ${content.ada.short}` : dialogTitles[DialogComponent.name]
```

is it better?

**Review by Crash-- — COMMENTED** [review]
With this PR, I realize that we're not screenshoting our CozyDialog on argos. We should fix that, no?

**Comment by JF-Cozy** [comment]
> With this PR, I realize that we're not screenshoting our CozyDialog on argos. We should fix that, no?

J'ai ajouté la première modale en screenshot... idéalement il faudrait peut-être screenshoter toutes les modales ?

**Comment by Crash--** [comment]
>idéalement il faudrait peut-être screenshoter toutes les modales ?

Idéalement notre script de screenshot sache jouer l'ensemble des variants de tous les composants ;)

**Review by Crash-- — APPROVED** [review]

**Pull request merged**